clients/src/main/java/org/apache/kafka/common/requests/AlterShareGroupOffsetsRequest.java

```diff
@@ -78,6 +79,25 @@ public static AlterShareGroupOffsetsRequest parse(Readable readable, short versi
         );
     }
 
+    public static AlterShareGroupOffsetsResponseData.AlterShareGroupOffsetsResponseTopic getErrorAlterShareGroup(
+        Errors error
+    ) {
+        return new AlterShareGroupOffsetsResponseData.AlterShareGroupOffsetsResponseTopic()
```

Comment:
In `DeleteShareGroupOffsetsResponse`, there are a top-level error code and message. These are missing in `AlterShareGroupOffsetResponse`. It would probably be best to add them, rather than setting an empty topic name and -1 partition index like this. wdyt?